django_stomp/helpers.py

```diff
@@ -115,3 +118,31 @@ def is_heartbeat_enabled(outgoing_heartbeat: int, incoming_heartbeat: int):
     More on: https://stomp.github.io/stomp-specification-1.1.html#Heart-beating
     """
     return outgoing_heartbeat > 0 and incoming_heartbeat > 0
+
+
+def set_ssl_connection(conn: StompConnection11) -> StompConnection11:
+    """Sets the SSL connection params given some values and return it"""
+
+    from django_stomp import settings
```

Comment:
Why do you import the module inside the method? Occurred circular import?

Reply:
Yes, yes. I put it because of the circular import.

Reply:
Usually this project imports from `django` lib.
https://github.com/juntossomosmais/django-stomp/blob/76ab17ad0017f8ddc950ef2894506d078e3a7e1b/django_stomp/builder.py#L5

Reply:
I changed the envvars to the method. could you check it out??